### PR TITLE
floodplain volume method

### PR DESCRIPTION
**Pull request by DirkEilander**
see #27 

TODO
- [x] update changelog
- [x] update docs
- [x] add test for `ucat_volume()` in tests/test_pyflwdir.py
- [ ] add manning roughness to ucat_volume method

### CODE REVIEW

File: pyflwdir/subgrid.py [hunk 1]
```diff
@@ -93,6 +93,55 @@ def segment_area(
...
 
 
+@njit
+def ucat_volume(
+    idxs_out,
+    idxs_ds,
+    seq,
+    hand,
+    area,
+    depths=np.arange(0.5, 3.0, 0.5, dtype=np.float32),
```
Comment (verseve) [inline pyflwdir/subgrid.py:103]: Why not have as default a depth profile interval of 0.5 m?

**Review by verseve — CHANGES_REQUESTED** [review]
Looks good! I only have one small question.

File: pyflwdir/subgrid.py (hunk 1 above)
Comment (DirkEilander) [reply pyflwdir/subgrid.py:103]: I think that's the case, isn't it?

**Comment by DirkEilander** [comment]
@verseve The last item on todo list for this PR is to add get manning roughness values associated with different flood depths. What input does Wflow.jl require? Is this the mean roughness for the entire floodplain at each level or just the mean roughness for that depth interval?

File: pyflwdir/subgrid.py (hunk 1 above)
Comment (verseve) [reply pyflwdir/subgrid.py:103]: Yes, it is indeed the case!

**Pull request merged**